specification/introduction/introduction.tex

```diff
@@ -8,73 +8,69 @@ \section{Design principles}
 defined in the section \ref{sec:transfer_prioritization}.
-Within each priority level, different types of transfers and different data types are
-prioritized in a well-defined deterministic manner.
+Within each priority level different types of transfers and different data types are
+prioritized in a well-defined, deterministic manner.
 
-The list of transport protocols supported by UAVCAN is provided in the chapter \ref{sec:transport_layer}.
+The list of transport protocols supported by UAVCAN is provided in chapter \ref{sec:transport_layer}.
 Non-redundant, doubly-redundant and triply-redundant transports are supported.
 More information on the physical layer and standardized physical connectivity options
 is provided in the chapter \ref{sec:physical_layer}.
 
+\mbox{\textbf{TODO}: add more about standard application capabilities when this section is complete.} 
+
 \section{Maintenance of the standard data type set}
 
-The UAVCAN maintainers are charged with advancing the standard data type set based on the input from adopters.
+The UAVCAN maintainers are charged with advancing the standard data type set based on input from adopters.
 This feedback is gathered via the official discussion
-forum\footnote{Please refer to \href{http://uavcan.org}{uavcan.org}.},
-which is open to everyone.
-
-The set of standard data type definitions is an important part of the specification
-(the data structure description language (DSDL) and related concepts are described in section \ref{sec:dsdl}).
-With minor exceptions explained in the chapter \ref{sec:application_layer},
-none of the standard data types are required to be supported by protocol implementations.
-Rather, their objective is to create a standardized data exchange environment
-allowing COTS\footnote{Commercial off-the-shelf equipment.} equipment manufactured by different vendors to
-interoperate.
-Non-COTS applications may avoid any dependency on the standard data type definitions (barring the aforementioned
-minor exceptions), relying solely on vendor-specific data types instead.
+forum\footnote{Please refer to \href{http://uavcan.org}{uavcan.org}.}
+which is open to the general public.
+
+The set of standard data type definitions\footnote{The Data Structure Description Language (DSDL) and related concepts are described in section \ref{sec:dsdl}.}
+is an integral part of the specification, however, there is only a small set of required data types needed to implement the protocol.
+A larger set of optional data types are defined to create a standardized data exchange environment
+supporting the interoperability of COTS\footnote{Commercial off-the-shelf equipment.} equipment manufactured by different vendors.
+While the maintainers are charged with maintining interoperability they will also accommodate custom data types defined for closed systems to be used in place of the provided types.
```

Comment:
`maintining` (I use VSCode with `ban.spellright`, works great with LaTeX)

> While the maintainers are charged with maintining interoperability they will also accommodate custom data types defined for closed systems to be used in place of the provided types.

This might be misinterpreted as if we're to allow inclusion of application-specific or vendor-specific data types into the standard set. I suggest removing this sentence completely.

Reply:
👌